cEP-0020.md

```diff
@@ -0,0 +1,389 @@
+on point thresholds. As users earn points, they move up an incremental
+series of levels. To infuse the level with meaning i.e.: a ranking within the
+system, the levels should be named in a way that indicates status.
+
+But the levels wouldn't be enough motivation to go until newcomers get
+some special privileges after completing each level:
+
+- Level-I:
+  - Name: Newbie
+  - Points_required = 5
+  - Privileges:
+    - Get assigned to a newcomer issue
+- Level-II:
+  - Name: Beginner
+  - Points_required = 15
+  - Privileges:
+    - Get assigned to a difficulty low issue
+- Level-III:
+  - Name: Intermediate
+  - Points_required = 30
+  - Privileges:
+    - Mark your own PR as wip
+- Level-IV:
+  - Name: Cool
+  - Points_required = 50
+  - Privileges:
+    - Mark a PR by another newcomer as wip
+- Level-V
+  - Name: Awesome
+  - Points_required = 100
+  - Privileges
+    - Get assigned to difficulty medium issue
+- Level-VI
+  - Name: Cool and Awesome
+  - Points_required = 200
+  - Privileges
+    - Assign newcomers to difficulty newcomer issue
+    - Assign newcomers to difficulty low issue
+- Level-VII
+  - Name: Expert and Trusted
+  - Points_required = 350
+  - Privileges
+    - Invite newcomers to org
+    - Assign newcomers to difficulty medium issue
+
+#### Note
+
+Implementing the above privileges is beyond the scope of the gamification system.
+The gamification system will merely expose a list of privileges that each
+person has, and other systems(i.e.: [corobo](https://github.com/coala/corobo))
+are expected to adapt to use these privileges.
+
+While implementing this process, a first manual check should be done for testing
+purpose, and then automated only if there is no serious risk or there is a good
+risk reaction mechanism in place.
+
+### Badges
+
+Another way of explicitly nudging a newcomer to action is to award badges for
+completing tasks. Badges enable newcomers to follow their performance and compare
+to other newcomers.
+Badges will be awarded after performing certain activities by the newcomers:
+
+- Badge-I:
+  - Name: The bug finder
+  - Details: The one who find bugs in the existing codebase
+  - Activities:
+    - Create at least one issue which labeled as bug
+      and approved by a maintainer i.e.: had label `process/verified`
+- Badge-II:
+  - Name: The bear hunter
+  - Details: The one who create issues about new bears
+  - Activities:
+    - Create at least one bear proposal issue that also had label
+      `process/varified`
+- Badge-III:
+  - Name: The bear writer
+  - Details: The one who write new bears
+  - Activities:
+    - Merged at least 2 bear proposal PRs
+- Badge-IV:
+  - Name: The helper
+  - Details: The one who answers every newcomer questions on Gitter
+  - Activities:
+    - At least 20 good messages to newcomer
+- Badge-V:
+  - Name: The coala Developer
+  - Details: The ones who are in the coala developers team
+  - Activities:
+    - Introduce `.coafile` in other projects
```

Comment:
I see that http://api.coala.io/en/latest/Developers/Newcomers_Guide.html says `run coala on a project of yours`. Is there any reason for the change here?

Also `Introduce .coafile` sounds a bit informal/not quite clear.